src/routes/index.tsx

```diff
@@ -119,6 +121,10 @@ const routes: RouteObject[] = [
       },
     ],
   },
+  { path: 'invite', element: <Invitation /> },
+  { path: 'error-400', element: <ErrorPage code={400} /> },
```

Comment:
plz double check on google (stackoverflow) how to use routes for such errors (e.g. best practices to display errors 400, 500)

Reply:
Done.